utils/backscrub.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+VERSION=20241106140722
+
+##
+## https://github.com/floe/backscrub
+##
+## backscrub, Fedora build script
+## Roman E. Chechnev <m0r1k>
+##
+
+ROOT_DIR=~/Downloads/backscrub_patched/
```

Comment:
This should be something the user can specify. Not everyone has that downloads folder called this way …

Reply:
ahah